src/fragments/lib/datastore/android/advanced-workflows/custom-primary-key/delete-snippet.mdx

```diff
@@ -0,0 +1,145 @@
+### Deleting records with Custom Primary Keys:
+
+In addition to deleting by the model type (or `Predicates.ALL`), a record with a custom primary key can be deleted the following ways:
+
+
+With a predicate:
+
+Custom Primary Key
+
+<Block name="Custom Primary Key">
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+Amplify.DataStore.query(Book.class, Where.identifier(Book.class,
+                                                      Where.identifier(Book.class, "12345")),
+                matches -> {
+                    if (matches.hasNext()) {
+                        Book book = matches.next();
+                        Amplify.DataStore.delete(book,
+                                deleted -> Log.i("MyAmplifyApp", "Deleted a book."),
+                                failure -> Log.e("MyAmplifyApp", "Delete failed.", failure)
+                        );
+                    }
+                },
+                failure -> Log.e("MyAmplifyApp", "Query failed.", failure));
+
+```
+
+</Block>
+<Block name="Kotlin - Callbacks">
+
+```kotlin
```

Comment:
@renebrandel - Android follows a different pattern from the other platforms re: PK vs PK + SK snippets, and I do agree with the current approach to instead show snippets for Java, Kotlin callback, Kotline roroutines, and RxJava, but wanted to get your sign off on it.